packages/shared/src/types/targeting/shared.ts

```diff
@@ -33,3 +34,26 @@ export interface PurchaseInfo {
     userType: purchaseInfoUser;
     product: purchaseInfoProduct;
 }
+
+export const abandonedBasketSchema = z.object({
+    amount: z.union([z.number(), z.literal('other')]),
+    billingPeriod: z.union([z.literal('ONE_OFF'), z.literal('MONTHLY'), z.literal('ANNUAL')]),
+    product: z.union([
+        z.literal('Contribution'),
+        z.literal('SupporterPlus'),
+        z.literal('SubscriptionProduct'),
```

Comment:
`SubscriptionProduct` is actually a union of several product names -
https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/productPrice/subscriptions.ts#L24